<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureGraphicsAdvanced</class>
 <widget class="QWidget" name="ConfigureGraphicsAdvanced">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="accessibleName">
   <string>Advanced</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_1">
       <property name="title">
        <string>Advanced Graphics Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="gpu_accuracy_layout" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_gpu_accuracy">
             <property name="text">
              <string>Accuracy Level:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="gpu_accuracy">
             <item>
              <property name="text">
               <string notr="true">Normal</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">High</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">Extreme(very slow)</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="async_present">
          <property name="text">
           <string>Enable asynchronous presentation (Vulkan only)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="renderer_force_max_clock">
          <property name="toolTip">
           <string>Runs work in the background while waiting for graphics commands to keep the GPU from lowering its clock speed.</string>
          </property>
          <property name="text">
           <string>Force maximum clocks (Vulkan only)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="async_astc">
          <property name="toolTip">
           <string>Enables asynchronous ASTC texture decoding, which may reduce load time stutter. This feature is experimental.</string>
          </property>
          <property name="text">
           <string>Decode ASTC textures asynchronously (Hack)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="use_reactive_flushing">
          <property name="toolTip">
           <string>Uses reactive flushing instead of predictive flushing. Allowing a more accurate syncing of memory.</string>
          </property>
          <property name="text">
           <string>Enable Reactive Flushing</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="use_asynchronous_shaders">
          <property name="toolTip">
           <string>Enables asynchronous shader compilation, which may reduce shader stutter. This feature is experimental.</string>
          </property>
          <property name="text">
           <string>Use asynchronous shader building (Hack)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="use_fast_gpu_time">
          <property name="toolTip">
           <string>Enables Fast GPU Time. This option will force most games to run at their highest native resolution.</string>
          </property>
          <property name="text">
           <string>Use Fast GPU Time (Hack)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="use_vulkan_driver_pipeline_cache">
          <property name="toolTip">
           <string>Enables GPU vendor-specific pipeline cache. This option can improve shader loading time significantly in cases where the Vulkan driver does not store pipeline cache files internally.</string>
          </property>
          <property name="text">
           <string>Use Vulkan pipeline cache</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="af_layout" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_1">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="af_label">
             <property name="text">
              <string>Anisotropic Filtering:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="anisotropic_filtering_combobox">
             <item>
              <property name="text">
               <string>Automatic</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Default</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2x</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4x</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8x</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>16x</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
